news_items/021_Json_schema_comment/index.html

```diff
@@ -0,0 +1,41 @@
+<style>
```

Comment:
Please rename the news item directory to use a lower-case `j`

Reply:
resolved in 502ad59f3